guides/common/modules/proc_registering-hosts.adoc

```diff
@@ -90,6 +90,7 @@ If set to `Yes`, public SSH keys will be installed on the registered host.
 The inherited value is based on the `host_registration_remote_execution` parameter.
 It can be inherited, for example from a host group, an operating system, or an organization.
 When overridden, the selected value will be stored on host parameter level.
+ifdef::client-content-dnf[]
```

Comment:
I don't think this has to do with RPM content, but rather a RHEL client system. For me RPM content really is the Katello RPM content feature, but this part also applies to just Foreman. AFAIK all RHEL systems can be registered directly to https://cloud.redhat.com/ even without Katello and the rh-cloud plugin.

The source code says the same thing, because there's no `plugin_present?('katello')`: https://github.com/theforeman/foreman/blob/2bc0afba704a41e50a5ec30356f53ab97fbbf40b/app/views/unattended/provisioning_templates/host_init_config/host_init_config_default.erb#L68-L71

Reply:
Yes, this is exactly what I want to bring upstream next: conditions for client operating systems so that we can differentiate between "client consumes Yum content" and "client runs RHEL".

For now, this is an improvement for downstream for me.

Reply:
I see no harm in this. From what I understand, this does not impact other than orcharhino documentation, so if it's an improvement, I say go for it.